src/Modules/Trading/Containers/trade.jsx

```diff
@@ -1,18 +1,30 @@
-import PropTypes             from 'prop-types';
-import React                 from 'react';
-import ChartLoader           from 'App/Components/Elements/chart-loader.jsx';
-import UILoader              from 'App/Components/Elements/ui-loader.jsx';
-import { connect }           from 'Stores/connect';
-import PositionsDrawer       from 'App/Components/Elements/PositionsDrawer';
+import PropTypes            from 'prop-types';
+import React                from 'react';
+import ChartLoader          from 'App/Components/Elements/chart-loader.jsx';
+import UILoader             from 'App/Components/Elements/ui-loader.jsx';
+import { connect }          from 'Stores/connect';
 import MarketIsClosedOverlay from 'App/Components/Elements/market-is-closed-overlay.jsx';
```

Comment:
alignment here

Reply:
@easteregg please sort the imports alphabetically.